### ISSUE TITLE
Reflect upload_status FAILURE in front-end.

### ISSUE THREAD

**Issue by phraenquex** [m2ms/fragalysis-frontend#1019]
Remaining issue from #995, which already moved to production.

**Comment by boriskovar-m2ms** [comment]
I'm not getting any information about the upload, only about the job execution. Currently we don't get any computed set for given job execution so the 'Open' button is grayed out because there is nothing to navigate to. If we add a new job status (UPLOAD_FAILED or something) then all I need to do is to specify a new status color for a job execution triangle in the project navigator and that is trivial. Not trivial to test right now because we have no way to reproduce the error. Somewhere is a discussion (which I can't find) about adding a job that fails so it would be useful to create a new job or parametrize the former to simulate this kind of failure.

**Comment by phraenquex** [comment]
Does it need changes to backend, to provide you with that information?  Or can you work out yourself whether the upload has failed?

If it needs backend, please add appropriat label, and assign to Alan.

The dud-job was added by @tdudgeon, described in #995 .

**Comment by boriskovar-m2ms** [comment]
Yes it needs back-end to work properly. I can try to infer that there is a high probability that upload has failed and might not be that easy but I think it should be quite easy to add this at the back-end side (just my assumption @alanbchristie please confirm) and very easy to use by front-end.
